tests/content.test.ts

```diff
@@ -32,11 +36,12 @@ describe(`MDC Formatter`, async () => {
         .filter(line => !line.includes('Code block style'))
         .filter(line => !line.includes('Lists should be surrounded by blank lines'))
 
+        // npm warnings, unrelated, e.g. 'npm warn config optional Use `--omit=optional` to exclude optional dependencies...'
+        .filter(line => !line.includes('npm warn config'))
```

Comment:
There was another error being exposed in the Vitest console related to running `npx` so this just silences the warning for the test.